XRTK-Core/Packages/com.xrtk.core/Runtime/Definitions/Utilities/Handedness.cs

```diff
@@ -11,12 +11,12 @@ namespace XRTK.Definitions.Utilities
     /// "Other" defines potential controllers that will offer a "third" hand, e.g. a full body tracking suit.
     /// </summary>
     [Flags]
-    public enum Handedness : byte
+    public enum Handedness
```

Comment:
Fixes problems with trying to set the value in the editor to `Any` or `Everything`

Reply:
Need to pull this out